src/Navigation/styles.css

```diff
@@ -0,0 +1,43 @@
+.Navigation {
+  min-height: 10vh;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  font-size: 26px;
+  color: white;
+  margin-bottom: 20px;
+}
+
+.Navigation-links {
+  display: flex;
+  flex-direction: row;
+  padding: 10px 20px;
+}
+
+.Navigation-mobile {
+  display: none;
+}
+
+@media (max-width: 996px) {
+  .Navigation {
+    flex-direction: column;
+  }
+  .Navigation-links {
+    flex-direction: column;
+  }
+  .Navigation-mobile {
+    display: none;
```

Comment:
You should use additional class to increase selector specificity. 
 .Navigation-mobile.MuiSvgIcon-root {
    display: none;
  }
  
  Documentation: https://mui.com/material-ui/api/icon/